test/fit_tests/deploy/rackhd_package_install.py

```diff
@@ -14,72 +27,111 @@
-        # install dependencies
-        self.assertEqual(fit_common.remote_shell(ENVVARS + "apt-get -y install rabbitmq-server")['exitcode'], 0, "rabbitmq-server Install failure.")
-        self.assertEqual(fit_common.remote_shell(ENVVARS + "apt-get -y install mongodb")['exitcode'], 0, "mongodb Install failure.")
-        self.assertEqual(fit_common.remote_shell(ENVVARS + "apt-get -y install snmp")['exitcode'], 0, "snmp Install failure.")
-        self.assertEqual(fit_common.remote_shell(ENVVARS + "apt-get -y install ipmitool")['exitcode'], 0, "ipmitool Install failure.")
-        # install Node
-        fit_common.remote_shell(ENVVARS + 'apt-get -y remove nodejs nodejs-legacy')
-        fit_common.remote_shell(ENVVARS +
-                                'wget --quiet -O - https://deb.nodesource.com/gpgkey/nodesource.gpg.key | sudo apt-key add -;'
-                                'echo "deb https://deb.nodesource.com/node_4.x trusty main" | tee /etc/apt/sources.list.d/nodesource.list;'
-                                'echo "deb-src https://deb.nodesource.com/node_4.x trusty main" | tee -a /etc/apt/sources.list.d/nodesource.list;')
-        fit_common.remote_shell(ENVVARS + 'apt-get -y update;'
-                                'apt-get -y install nodejs;'
-                                )
-        fit_common.remote_shell('apt-get -y install npm;npm config set https-proxy '
-                                + fit_common.GLOBAL_CONFIG['repos']['proxy'])
-        # Install Ansible
-        self.assertEqual(fit_common.remote_shell(ENVVARS + "apt-get -y install ansible")['exitcode'], 0, "ansible Install failure.")
-        self.assertEqual(fit_common.remote_shell(ENVVARS + "apt-get -y install apt-mirror")['exitcode'], 0, "apt-mirror Install failure.")
-        self.assertEqual(fit_common.remote_shell(ENVVARS + "apt-get -y install amtterm")['exitcode'], 0, "amtterm Install failure.")
-        self.assertEqual(fit_common.remote_shell(ENVVARS + "apt-get -y install isc-dhcp-server")['exitcode'], 0, "isc-dhcp-server Install failure.")
+        # update sudoers to preserve proxy environment
+        sudoersproxy = open("sudoersproxy", 'w')
+        sudoersproxy.write('Defaults env_keep="HOME no_proxy http_proxy https_proxy"\n')
+        sudoersproxy.close()
+        fit_common.remote_shell('pwd')
+        fit_common.scp_file_to_ora("sudoersproxy")
+        self.assertEqual(fit_common.remote_shell('cp sudoersproxy /etc/sudoers.d/'
+                                                  )['exitcode'], 0, "sudoersproxy config failure.")
+        os.remove('sudoersproxy')
+        # install git
+        self.assertEqual(fit_common.remote_shell(ENVVARS + "apt-get -y install git")['exitcode'], 0, "Git install failure.")
+        self.assertEqual(fit_common.remote_shell(ENVVARS + "apt-get -y update")['exitcode'], 0, "update failure.")
+        self.assertEqual(fit_common.remote_shell(ENVVARS + "apt-get -y dist-upgrade")['exitcode'], 0, "upgrade failure.")
```

Comment:
This is the sequence from the RackHD install documentation. I'd love to get rid of it.